java/j2ee.eclipselink/nbproject/project.properties

```diff
@@ -38,3 +38,5 @@ release.external/jakarta.persistence-2.2.3.jar=modules/ext/eclipselink/jakarta.p
 release.build/jakarta.persistence-2.2.3-doc.zip=modules/ext/docs/jakarta.persistence-2.2.3-doc.zip
 
 extra.module.files=modules/ext/docs/jakarta.persistence-2.2.3-doc.zip
+
+sigtest.public.packages=org.eclipse,javax.persistence
```

Comment:
Strange error here when generating signature file.

Reply:
Have to check it through, this one could be just used as another Ant library, in that case this shall not offer a public API.